server/container_remove.go

```diff
@@ -12,42 +9,11 @@ import (
 // RemoveContainer removes the container. If the container is running, the container
 // should be force removed.
 func (s *Server) RemoveContainer(ctx context.Context, req *pb.RemoveContainerRequest) (*pb.RemoveContainerResponse, error) {
-	logrus.Debugf("RemoveContainerRequest %+v", req)
-	c, err := s.GetContainerFromRequest(req.ContainerId)
+	_, err := s.ContainerServer.Remove(req.ContainerId, true)
```

Comment:
ContainerRemove().  RemoveContainer() already existed so I couldn't use that.  I do need to change it though